### ISSUE TITLE
Ideas for Chat Experiment's second blog post

### ISSUE THREAD

**Issue by AI-DennisNdegwa** [MozillaFoundation/digital-skills-observatory#276]
Brain dump and random thoughts that will help in the making of the second blog for this prototype. Anyone, and especially the actors greatly encouraged to drop random thoughts.

**Comment by AI-DennisNdegwa** [comment]
Does the lack of internet connection affordability prevent users from trying new things OR does the lack of knowledge of utilizable internet and app features make users budget lower for data usage?

**Comment by AI-DennisNdegwa** [comment]
Just because you offer to teach someone something it doesn't necessarily mean that they will always be interested in learning it. More so if there is little perceived value of whatever you want to teach them. Changing their perception first, by demonstrating the value of such knowledge is the best approach in most cases.

**Comment by Uwanja** [comment]
During the interaction the respondents engaged Sterro in bouts. This could be attributed to the time they renew their data bundles and also to the fact that most were working and they could afford to chat during lunch hour, in the evening past 5PM or early morning before they got to work at 8am.

Some respondents asked Sterro non techy questions out of the scope of Information and Technology as one would expect given the way Sterro's persona  was given to the participants during on boarding.

Some people were afraid to try some new things in their phone even if it wouldn't cost them anything like finding the Android Easter egg.

**Comment by secretrobotron** [comment]
👍 this is great stuff, guys. I'll add some as well based on the analysis. Do you want to continue compiling thoughts to publish after Sterro goes offline (😢)? Or before?

**Comment by AI-DennisNdegwa** [comment]
I think we can keep gathering thoughts for the remainder of Sterro. At the end of it there could be useful bits that could be included in a final blog/report.

**Comment by AI-DennisNdegwa** [comment]
Participants are more likely to learn new things when solving personal problems. Teaching a solution to a problem that may or may not occur in the future doesn't always encourage participation and interest.

**Comment by secretrobotron** [comment]
Two more considerations:
- @WinnieMakokha's comments in https://github.com/MozillaFoundation/digital-skills-observatory/issues/277#issuecomment-258087526
- Continuing Sterro as a community project, like https://github.com/MozillaFoundation/digital-skills-observatory/issues/278

**Comment by AI-DennisNdegwa** [comment]
One can observe that equipping people with digital skills gives some individuals the ability to infringe on other people's rights. E.g keeping track of their spouse's location history, writing controversial blog posts, accessing a lover's/spouse's WhatsApp account remotely via WhatsApp web, tricking an unskilled person into giving you access to their email account and consequently their MPESA statements. e.t.c

**Comment by denomain** [comment]
One can observe that equipping people with digital skills gives some individuals the ability to infringe on other people's rights. E.g keeping track of their spouse's location history, writing controversial blog posts, accessing a lover's/spouse's WhatsApp account remotely via WhatsApp web, tricking an unskilled person into giving you access to their email account and consequently their MPESA statements. e.t.c

**Comment by secretrobotron** [comment]
@denomain wrote some good thoughts into a google doc a little while ago. Considering we're wrapping up report-writing, do you guys want to publish a final blog post? Maybe speculate on a couple of ideas for [the future of Sterro](https://github.com/MozillaFoundation/digital-skills-observatory/issues/278)?

**Comment by AI-DennisNdegwa** [comment]
@secretrobotron do you mean this one https://docs.google.com/document/d/10CuCpS7vMUdLJA2XL0roFdUtqkqDh-ILNLv6i8r4lJM/edit ? I could add some more flesh to it if it is something that would be helpful somewhere at this point.

**Comment by secretrobotron** [comment]
@denomain as an artifact of the project and of Mozilla's community approach, I think it would be great to publish. The other guys were going to add to it as well when we last spoke about it, right? Do you know if they're still interested?

**Comment by AI-DennisNdegwa** [comment]
@secretrobotron Yes. That was the plan. We were supposed to do a collective write-up. @Uwanja @Josiahma1 you guys got some time to check it out and add some stuff? Might be good for publishing a blog post on mozilla-kenya.org as well.
